app/backend/app.py

```diff
@@ -69,8 +53,8 @@ async def assets(path):
 # can access all the files. This is also slow and memory hungry.
 @bp.route("/content/<path>")
 async def content_file(path):
-    blob_container = current_app.config[CONFIG_BLOB_CLIENT].get_container_client(AZURE_STORAGE_CONTAINER)
```

Comment:
I now store the specific container client in the config, since we seem to always use that anyway (versus the more general storage client).